iOS_Concepts.playground/Contents.swift

```diff
@@ -1,3 +1,34 @@
-import UIKit
+import Foundation
+
+//extensions to add functionality to existing types.
+protocol Houses {
```

Comment:
Naming of protocols, structs, classes, arrays, functions, variables etc is very important. In this instance `Houses` usually refers to a list of items eg. an array so it can be confusing when working with a team.